src/templates/organization.js

```diff
@@ -105,12 +106,15 @@ export default function OrganizationTemplate({ data }) {
             <Tags org={org} />
 
             {org.fullPhotos[0] && (
-              <div className="carousel my-8 bg-gray-200 rounded-lg p-4">
-                <img
-                  src={org.fullPhotos[0].publicURL}
-                  alt={org.title}
-                  style={{ height: "20rem" }}
-                  className="organization-img mx-auto"
+              <div className="carousel my-8 bg-gray-200 rounded-lg">
+                <Carousel
+                  height="20rem"
+                  // FIXME: using reverse since the images are in the reverse order of airtable
```

Comment:
I do not know why but the org photos are in the reverse order of the display on airtable (https://airtable.com/shrbEdONg9Mmp2Tnf/tblPH0ZgLZYGVEhEo?blocks=hide).

Does someone know why? Which behavior should we use?

Reply:
hmm, lets use the airtable order, but i don't think it makes a big difference :)

Reply:
I will do that, any idea of the reason why it is reversed ?

Reply:
@loiclefloch I believe it's related to this breaking change that Airtable made:

https://support.airtable.com/hc/en-us/articles/216771888

Reply:
@loiclefloch reading that doc page, I wonder if we just want to toggle that option on Airtable and avoid the hack here in Gatsby. It'd break all the orders once but we'd never have to worry about this again. Thoughts @jasonLaster ?